cmd/core/main.go

```diff
@@ -103,17 +105,21 @@ func main() {
 		"controller shared informer lister full re-sync period")
 	flag.StringVar(&oam.SystemDefinitonNamespace, "system-definition-namespace", "vela-system", "define the namespace of the system-level definition")
 	flag.IntVar(&controllerArgs.ConcurrentReconciles, "concurrent-reconciles", 4, "concurrent-reconciles is the concurrent reconcile number of the controller. The default value is 4")
+	flag.Float64Var(&qps, "kube-api-qps", 50, "the qps for reconcile clients. Low qps may lead to low throughput. High qps may give stress to api-server. Raise this value if concurrent-reconciles is set to be high.")
+	flag.IntVar(&burst, "kube-api-burst", 100, "the burst for reconcile clients. Recommend setting it qps*2.")
 	flag.DurationVar(&controllerArgs.DependCheckWait, "depend-check-wait", 30*time.Second, "depend-check-wait is the time to wait for ApplicationConfiguration's dependent-resource ready."+
 		"The default value is 30s, which means if dependent resources were not prepared, the ApplicationConfiguration would be reconciled after 30s.")
 	flag.StringVar(&controllerArgs.OAMSpecVer, "oam-spec-ver", "v0.3", "oam-spec-ver is the oam spec version controller want to setup, available options: v0.2, v0.3, all")
 	flag.StringVar(&pprofAddr, "pprof-addr", "", "The address for pprof to use while exporting profiling results. The default value is empty which means do not expose it. Set it to address like :6666 to expose it.")
+	flag.BoolVar(&commonconfig.PerfEnabled, "perf-enabled", false, "Enable performance logging for controllers, disabled by default.")
 
 	flag.Parse()
 	// setup logging
 	klog.InitFlags(nil)
 	if logDebug {
-		_ = flag.Set("v", strconv.Itoa(int(commonconfig.LogDebug)))
+		_ = flag.Set("v", strconv.Itoa(commonconfig.LogDebug))
 	}
+	klog.InfoS("configs", "syncPeriod", syncPeriod.Seconds(), "qps", qps, "burst", burst, "concurrent-reconciles", controllerArgs.ConcurrentReconciles)
```

Comment:
If this is the only one place where we use klog, and in other places, it just uses `klog.KObj()`, why just remove all klog in kubevela. Two styles of logger may result in confusion and abusing.

Reply:
Ok, I will fix it for other handlers.